WinUIGallery/Controls/DesignGuidance/TypographyControl.xaml

```diff
@@ -137,22 +138,13 @@
                     VerticalAlignment="Center"
                     IsTextSelectionEnabled="True"
                     Text="{x:Bind ResourceName}" />
-                <Grid
-                    Width="36"
-                    Height="36"
-                    Margin="4,2,8,0">
-                    <Button
+                <local:CopyButton
                         x:Name="CopyToClipboardButton"
-                        HorizontalAlignment="Stretch"
-                        VerticalAlignment="Stretch"
-                        AutomationProperties.Name="Copy Style to clipboard"
                         Click="CopyToClipboardButton_Click"
                         Content="&#xE8C8;"
-                        FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                        Style="{StaticResource SubtleButtonStyle}"
-                        ToolTipService.ToolTip="Copy Style to clipboard"
+                        Margin="4,2,8,0"
+                        ToolTipService.ToolTip="Copy to clipboard"
                         Visibility="Collapsed" />
```

Comment:
(correct me if i'm mistaken, but) This property makes it so that it still is hidden by default. That isn't consistent across all the examples: usually it's always visible.